### PR TITLE
Added in further details on where to find Observe documentation.

### PR DESCRIPTION
**Pull request by matthewcullum-gds**
Added in further details on where to find Observe documentation extra Observe specific details.

Single: matthewcullum-gds

### CODE REVIEW

File: source/documentation/observe-support.md [hunk 1]
```diff
@@ -1,3 +1,70 @@
...
+Paas information can be found at [Paas](https://docs.cloud.service.gov.uk/before_you_start.html#before-you-start).
+
+  - The spaces that are relevant to Observe are:- 
+    - prometheus-grafana (production grafana)
+    - prometheus-production
+    - prometheus-staging
+
+### Access to our secrets
+
+Our secrets and passwords are stored within [github](https://github.com/alphagov/re-secrets/tree/master/observe) our specific secrets are in re-secrets/observe.
```
Comment (JonathanHallam) [inline source/documentation/observe-support.md]: Not relevant to this pr but we should think about renaming our bit of the secrets repo

File: source/documentation/observe-support.md [hunk 2]
```diff
@@ -1,3 +1,70 @@
...
+[profile re-prometheus-dev]
+region = eu-west-1
+source_profile=fredbloggs
+role_arn=arn:aws:iam::931679966755:role/fred.bloggs-admin
+mfa_serial=arn:aws:iam::1234567890123:mfa/fred.bloggs@digital.cabinet-office.gov.uk
+```
+
+#### Production and Staging Stacks
+
+gds-prometheus-production = 455214962221
```
[the remark this answers is not in the document]
Comment (matthewcullum-gds) [reply source/documentation/observe-support.md]: We have posted on slack and discussed this within the team and the consensus is that these are not sensitive info.

File: source/documentation/observe-support.md [hunk 3]
```diff
@@ -1,3 +1,70 @@
...
+
+gds-prometheus-staging    = 027317422673
+
+example .aws/config
+```
+[profile staging]
+region = eu-west-1
+source_profile=fredbloggs
+role_arn=arn:aws:iam::027317422673:role/Administrator
+mfa_serial=arn:aws:iam::1234567890123:mfa/matthew.cullum@digital.cabinet-office.gov.uk
```
[the remark this answers is not in the document]
Comment (matthewcullum-gds) [reply source/documentation/observe-support.md]: Missed those thanks - updated

File: source/documentation/observe-support.md [hunk 4]
```diff
@@ -1,3 +1,70 @@
...
+
+[profile production]
+region = eu-west-1
+source_profile=fredbloggs
+role_arn=arn:aws:iam::455214962221:role/Administrator
+mfa_serial=arn:aws:iam::1234567890123:mfa/matthew.cullum@digital.cabinet-office.gov.uk
```
[the remark this answers is not in the document]
Comment (matthewcullum-gds) [reply source/documentation/observe-support.md]: Updated thanks

File: source/documentation/prometheus-for-paas-support.md [hunk 5]
```diff
@@ -1,3 +1,72 @@
...
+
+#### Production and Staging Stacks
+
+gds-prometheus-production = 455214962221
+
+gds-prometheus-staging    = 027317422673
+
+example .aws/config
+```
+[profile staging]
```
Comment (idavidmcdonald) [inline source/documentation/prometheus-for-paas-support.md]: Optional: Profile names could be more consistent with the re-prometheus-dev example (so they are all `re-prometheus-env` or all just `env`).

File: source/documentation/prometheus-for-paas-support.md [hunk 6]
```diff
@@ -1,3 +1,72 @@
...
+role_arn=arn:aws:iam::455214962221:role/Administrator
+mfa_serial=arn:aws:iam::1234567890123:mfa/fred.bloggs@digital.cabinet-office.gov.uk
+```
+
+#### AWS ECS Access
+Access to our EC2 instances is detailed within [EC2 Access](https://github.com/alphagov/prometheus-aws-configuration-beta).
+
+### Access to Paas
+
+Paas information can be found at [Paas](https://docs.cloud.service.gov.uk/before_you_start.html#before-you-start).
```
Comment (idavidmcdonald) [inline source/documentation/prometheus-for-paas-support.md]: If we are going to link to the PaaS here I would suggest either we link to the default index page of the docs (https://docs.cloud.service.gov.uk/) or instead we make it specific that a user will need to be added to the below spaces and link to the section on how to add a user to a space (https://docs.cloud.service.gov.uk/orgs_spaces_users.html#add-users-to-a-space)

File: source/documentation/prometheus-for-paas-support.md [hunk 7]
```diff
@@ -1,3 +1,72 @@
...
+    - prometheus-production
+    - prometheus-staging
+
+### Access to our secrets
+
+Our secrets and passwords are stored within [github](https://github.com/alphagov/re-secrets/tree/master/observe) our specific secrets are in re-secrets/observe.
+
+### Access Restrictions
+
+Access to the Prometheis/Alertmanagers is controlled by ip whitelisting and basic auth.
```
Comment (idavidmcdonald) [inline source/documentation/prometheus-for-paas-support.md]: The story I'm working on has just changed this a little.

Prometheus is accessible by office IP safelisting and then falls back to basic auth.

Alertmanager is accessible by office IP safelisting but no longer has a fall back to basic auth.

File: source/documentation/prometheus-for-paas-support.md [hunk 8]
```diff
@@ -1,3 +1,74 @@
...
```
Comment (JonathanHallam) [inline source/documentation/prometheus-for-paas-support.md]:
```suggestion
Prometheus is accessible by office IP safelisting and then falls back to basic auth.
Alertmanager is accessible by office IP safelisting but no longer has a fall back to basic auth.
```

**Pull request merged**